pkg/webhook/operatingsystemconfig/ensurer_test.go

```diff
@@ -33,27 +32,7 @@ import (
 	"github.com/gardener/gardener-extension-shoot-rsyslog-relp/pkg/apis/rsyslog"
 	rsysloginstall "github.com/gardener/gardener-extension-shoot-rsyslog-relp/pkg/apis/rsyslog/install"
 	. "github.com/gardener/gardener-extension-shoot-rsyslog-relp/pkg/webhook/operatingsystemconfig"
-)
-
-var (
-	//go:embed testdata/60-audit.conf
-	rsyslogConfig []byte
-	//go:embed testdata/60-audit-with-tls.conf
-	rsyslogConfigWithTLS []byte
-
-	//go:embed testdata/configure-rsyslog.sh
-	confiugreRsyslogScript []byte
-	//go:embed testdata/process-rsyslog-pstats.sh
-	processRsyslogPstatsScript []byte
-
-	//go:embed testdata/00-base-config.rules
-	baseConfigRules []byte
-	//go:embed testdata/10-privilege-escalation.rules
-	privilegeEscalationRules []byte
-	//go:embed testdata/11-privileged-special.rules
-	privilegeSpecialRules []byte
-	//go:embed testdata/12-system-integrity.rules
-	systemIntegrityRules []byte
+	testcommon "github.com/gardener/gardener-extension-shoot-rsyslog-relp/test/common"
```

Comment:
It seems a bit weird to import a package from the `github.com/gardener/gardener-extension-shoot-rsyslog-relp/test` folder here. It should probably by the other way around. Let's discuss this offline.

Reply:
Agree. It is not recommended to use in production code helpers from a test code. It should be rather the other way around - test code to import production vars/constants.